Header header

# The size of the array
uint32 size
# The array containing all the detections
Detection[] data
